tests/pytests/data/H0010_0023_SR2/ATNM_MEASURED_040101_050101_V03_0_sliced_-41001.xc

```diff
@@ -0,0 +1,64 @@
+DAFETF NAIF DAF ENCODED TRANSFER FILE
+'DAF/CK  '
+'2'
+'6'
+'MEX CK; MEASURED; OBTAINED FROM TELEMETRY                   '
+BEGIN_ARRAY 1 43
+'MEX MEASURED ATTITUDE                   '
+'124591B6082^B'
+'1245926CA7B^B'
+'-A029'
+'1'
+'3'
+'1'
+43
+'1C03A61D1A1F56^0'
+'3954C69BA9111^0'
+'13A448099CEEAC^0'
+'F7240734FA7588^0'
+'-31103EE98F678^-2'
+'-C34FB53FB4FF48^-3'
+'166E1B16079174^-2'
+'1C0B44C0011506^0'
+'394DC6801635AE^0'
+'13D16FBFDB96A2^0'
+'F7212E0033FB88^0'
+'-311A469EA0DD1C^-2'
+'-C0CF4CA4DD0DB8^-3'
+'16F36E10C67BBB^-2'
+'1C1FBA7FE21B3A^0'
+'393DB00036A74A^0'
+'143D2F40347883^0'
+'F719D9FFD4FBB8^0'
+'-31F41D6FA5CA76^-2'
+'-B7242AF349E57^-3'
+'1874AD00FD45B9^-2'
+'1C36C57FE29F83^0'
+'392EAE7FF7E06^0'
+'14AB06002EAAE2^0'
+'F71199FFDFB878^0'
+'-323107D041659A^-2'
+'-BF76CB16BA03A^-3'
+'17787AB7F90EA^-2'
+'1C3FD09D95C745^0'
+'3927D8C350B0BA^0'
+'14DA844A3C102C^0'
+'F70E2A5D9902F8^0'
+'-320C56FC7BB3A6^-2'
+'-C0D3BF88B07968^-3'
+'177878A400885^-2'
+'124591B6082^B'
+'124591D105E^B'
+'124592110BA^B'
+'1245925105DFFF^B'
+'1245926CA7B^B'
+'124591B6082^B'
+'1^1'
+'5^1'
+END_ARRAY 1 43
+TOTAL_ARRAYS 1
+ ~NAIF/SPC BEGIN COMMENTS~
+This CK is for testing with the image: /scratch/mex2020/H0010_0023_SR2.IMG.cub
+
+This CK was generated using the following command: {}
```

Comment:
Do we know if there is a way to have this filled in with the actual command?

Reply:
Not sure. None of the other ones have it, though.